### PR TITLE
Update changelog for 1.7.0 release

### LINKED ISSUE #7484: Release: v1.7.0
([Documentation](./docs/release-management.md))

- [x] Update changelog in `readme.txt` (#7485)
- [x] Create release candidate
- [x] Do some smoke-testing
- [x] Create stable release
- [ ] Approve release on WordPress.org in email notification
- [ ] Communicate widely!

### PR DESCRIPTION
**Pull request by swissspidy**
Fixes #7484

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -109,6 +109,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.7.0 =
+
+**Release Date:** May 11, 2021.
+
+* New feature: save individual pages as custom page templates.
+* Improvements to the pre-publish checklist.
+* UI improvements in the editor and dashboard.
+* Imprvements to the built-in templates.
```
Comment (dreamofabear) [inline readme.txt]: "Fixed animations in existing templates."?

File: readme.txt [hunk 2]
```diff
@@ -109,6 +109,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.7.0 =
+
+**Release Date:** May 11, 2021.
+
+* New feature: save individual pages as custom page templates.
+* Improvements to the pre-publish checklist.
```
Comment (dreamofabear) [inline readme.txt]: "Reduced noisiness in the pre-publish checklist."? 

"Adjusted triggering thresholds"?

File: readme.txt [hunk 3]
```diff
@@ -109,6 +109,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.7.0 =
+
+**Release Date:** May 11, 2021.
+
+* New feature: save individual pages as custom page templates.
```
Comment (dreamofabear) [inline readme.txt]: "New feature: save a single page as a template for reuse across all your stories."?

File: readme.txt [hunk 4]
```diff
@@ -109,6 +109,16 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
+* Improvements to the pre-publish checklist.
+* UI improvements in the editor and dashboard.
```
Comment (dreamofabear) [inline readme.txt]: Can we be more specific? Or consider removing (covered by last point).

File: readme.txt (hunk 1 above)
Comment (swissspidy) [reply readme.txt]: "Improvements to the built-in templates and their animations."?

AFAIK not just the animations were improved

File: readme.txt (hunk 2 above)
Comment (swissspidy) [reply readme.txt]: "Improvements to thresholds in the pre-publish checklist to reduce noise."?

**Review by dreamofabear — APPROVED** [review]

**Pull request merged**